src/ResourceManagement/PrivateDns/Domain/IARecordSet.cs

```diff
@@ -14,6 +14,6 @@ public interface IARecordSet :
         /// <summary>
         /// Gets the IP v4 addresses of A records in this record set.
         /// </summary>
-        IReadOnlyList<string> IPv4Adresses { get; }
```

Comment:
Should be I`p`v4 for naming consistency.

Reply:
We get from DNS with such kind of naming for IP related.